components/list/list-item.js

```diff
@@ -26,65 +29,104 @@ class ListItem extends RtlMixin(LitElement) {
 		const layout = css`
 			:host {
 				display: block;
-				margin: 1px 0;
+				margin: 0.05rem 0;
 			}
 			:host[hidden] {
 				display: none;
 			}
 			.d2l-list-item-flex {
 				display: flex;
+				position: relative;
 			}
 			.d2l-list-item-content {
+				background: var(--d2l-list-item-background, none);
 				border-bottom: var(--d2l-list-item-divider-bottom, 1px solid var(--d2l-color-mica));
 				border-top: var(--d2l-list-item-divider-top, 1px solid var(--d2l-color-mica));
 				box-sizing: content-box;
-				margin-bottom: -1px;
-				margin-top: -1px;
+				margin-bottom: -0.05rem;
+				margin-top: -0.05rem;
 				padding-bottom: var(--d2l-list-item-divider-padding-bottom, 0);
 				padding-top: var(--d2l-list-item-divider-padding-top, 0);
 				position: relative;
 				width: 100%;
 			}
+			.d2l-list-item-flex:hover .d2l-list-item-content {
+				background: var(--d2l-list-item-hover-background, none);
+			}
 			.d2l-list-item-content-flex {
 				display: flex;
 				flex-grow: 1;
 				justify-content: stretch;
-				margin: 18px 0;
-				padding: var(--d2l-list-item-content-padding, 0);
+				padding: 0.55rem var(--d2l-list-item-content-padding-side, 0);
 			}
-			::slotted([slot="illustration"]){
-				align-self: flex-start;
+			::slotted([slot="illustration"]) {
 				display: flex;
 				flex-grow: 0;
 				flex-shrink: 0;
+				margin: 0.15rem 0;
 				margin-right: 0.9rem;
-				max-height: 52px;
-				max-width: 90px;
+				max-height: 2.6rem;
+				max-width: 4.5rem;
 				overflow:hidden;
 			}
 			:host([dir="rtl"]) ::slotted([slot="illustration"]) {
 				margin-left: 0.9rem;
 				margin-right: 0;
 			}
-			:host([illustration-outside]) ::slotted([slot="illustration"]) {
-				margin-bottom: 18px;
-				margin-top: 18px;
-			}
 			::slotted([slot="actions"]) {
 				align-self: flex-start;
-				display: flex;
+				display: grid;
```

Comment:
What is the rationale for switching to grid - is it to achieve vertical centering of the input?  Are there issues in IE11 or Edge?

Reply:
It makes it better to add multiple actions that have the correct responsive behaviour. I'll need to test it in edge.

It shouldn't be vertically centering. So that is a mistake I'll have to fix.

It also makes spacing easier to define with the grid-gap.